modules/pruning-images.adoc

```diff
@@ -5,111 +5,152 @@
+      spec:
+        template:
+          spec:
+            restartPolicy: OnFailure
+            containers:
+            - image: "quay.io/openshift/origin-cli:4.1"
+              resources:
+                requests:
+                  cpu: 1
+                  memory: 1Gi
+              terminationMessagePolicy: FallbackToLogsOnError
+              command:
+              - oc
+              args:
+              - adm
+              - prune
+              - images
+              - --certificate-authority=/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
+              - --keep-tag-revisions=5
+              - --keep-younger-than=96h
+              - --confirm=true
+              name: image-pruner
+            serviceAccountName: pruner
 ----
 
-Currently, to prune images you must first log in to the CLI as a user with an
-access token. The user must also have the cluster role `system:image-pruner` or
-greater (for example, `cluster-admin`).
+Running the `oc adm prune images [<options>]` command.
 
+* To manually prune images that are no longer required by the system due to age, status, or exceed limits by running the following command:
++
+----
+$ oc adm prune images [<options>]
+----
++
 Pruning images removes data from the integrated registry unless
 `--prune-registry=false` is used.
-
++
 Pruning images with the `--namespace` flag does not remove images, only
-imagestreams. Images are non-namespaced resources. Therefore, limiting pruning
-to a particular namespace makes it impossible to calculate their current usage.
-
-By default, the integrated registry caches blobs metadata to reduce the number
-of requests to storage, and increase the speed of processing the request.
-Pruning does not update the integrated registry cache. Images pushed after
-pruning that contain pruned layers will be broken, because the pruned layers
-that have metadata in the cache will not be pushed. Therefore, it is necessary
-to clear the cache after pruning. This can be accomplished by redeploying the
-registry:
-
+imagestreams. Images are non-namespaced resources. Therefore, limiting pruning to a particular namespace makes it impossible to calculate their current usage.
++
+* By default, the integrated registry caches blobs metadata to reduce the number of requests to storage, and increase the speed of processing the request. Pruning does not update the integrated registry cache. Images pushed after pruning that contain pruned layers will be broken, because the pruned layers that have metadata in the cache will not be pushed. Therefore, it is necessary to clear the cache after pruning. This can be accomplished by redeploying the registry:
++
 ----
 $ oc rollout restart deployment/image-registry -n openshift-image-registry
 ----
-
++
 If the integrated registry uses a Redis cache, you must clean the database
 manually.
-
++
 If redeploying the registry after pruning is not an option, then you must
 permanently disable the cache.
-
++
```

Comment:
Should the route  be a prerequisite?

Reply:
I think it's covered by exposing the registry. :) Thanks!